setup.py

```diff
@@ -29,7 +29,8 @@
     "natsort >= 5.0.1",
     "psutil >= 4.4.2",
     "zipstream >= 1.1.4",
-    "WsgiDAV >= 3.0.0"
+    "WsgiDAV >= 3.0.0",
+    'dataclasses >= 0.8; python_version < "3.6.0"'
```

Comment:
``python_version < 3.6.0`` ? :thinking:

Reply:
That's indeed an error. It's new in Python 3.7, but has been backported via a pip package for older versions. It should be < 3.7.0.